### ISSUE TITLE
Select 2 Screen reader Ajax Issue

### ISSUE THREAD

**Issue by mattwebcoop** [select2/select2#6007]
Seeing an issue whereby screen reading software (NVDA in this case) is not reading out the drop down options when they are populated via AJAX, it just reads out the search term entered. This doesn't occur when the data is not populated via AJAX.

 I've tested this on 4.0.13 and the 4.1.0 Release Candidate. I've included below the code that initializes Select2 

```javascript
$('#' + dropdown).select2({
    allowClear: true,
    minimumInputLength: 3,
    quietMillis: 100,
    placeholder: 'Select Option',
    ajax: {
        url: '@Url.Action("GetAll", "Options")',
        type: "GET",
        dataType: 'json',
        delay: 100,
        data: function(params) {
            return {
                q: params.term,
                pageSize: 50,
                page: params.page
            };
        },                                    
        processResults: function (data) {
                                  
            for (let i = 1; i < 6; i++) {
                let ddl = "Options" + i + "Id";
                if (ddl != dropdown) {
                    let id = $('#' + ddl).val();
                    if (id != null) {
                        data = $.grep(data,
                            function(el) {
                                return (el.id !== id);
                            });
                    }
                }
            }

            return { results: data };
        },                        
    }          
});
```

**Comment by pramodDesign** [comment]
@mattwebcoop I have faced same kind issue with 4.0.13. Created a issue https://github.com/select2/select2/issues/6058

**Comment by Marika27** [comment]
@mattwebcoop 2 years later and I am facing the same issue as you with ajax. The screen reader does not read the option elements or the selected option. Did you manage to fix your issue?

**Comment by mattwebcoop** [comment]
@Marika27 We didn't fix the issue unfortunately, the client at the time was happy to just put it as a known issue.
